license-header.js

```diff
@@ -0,0 +1,15 @@
+module.exports.license = `/**
+* Copyright 2016-2021 Skyscanner Limited.
+*
```

Comment:
Should this be in here rather than provided as an arg?

Reply:
Could you elaborate on this? Not sure what you mean here